modules/ROOT/pages/api-auto-discovery-new-concept.adoc

```diff
@@ -1,34 +1,34 @@
-
-By configuring autodiscovery you can use policy management, and API Analytics, accessed from within API Manager. +
-When autodiscovery is correctly configured in your Mule application you can say that your application's API is _tracked by_ or _paired to_ API Manager.
-
-An API in a Mule runtime can only be associated with one autodiscovery instance. No two Autodiscoveries can be associated to the same API in a Mule Runtime.
+When autodiscovery is correctly configured in your Mule application, you can say that your application's API is _tracked by_ (green dot) or _paired to_ API Manager. You can associate an API in a Mule setup with only one autodiscovery instance at a given time. 
 
 == Prerequisites
 
-To configure autodiscovery for your Mule application you need to meet the following requirements:
+To configure autodiscovery for your Mule application, ensure that:
 
-* API must exist in API Manager, configured with either a basic endpoint, or a proxy endpoint.
-* Mule must be configured to use Anypoint Platform credentials.
-** The platform credentials give your application access to the API Configuration in API Manager. These credentials must be set before starting the Mule runtime that executes your application. +
-See xref:org-credentials-config-mule4.adoc[Configuring Organization Credentials in Mule Runtime 4] or xref:org-credentials-config-mule3.aroc[Configuring Organization Credentials in Mule Runtime 3] for more details.
-* The autodiscovery element must be configured in your Mule application. This XML element must point to the specific API in API manager which you want to pair to.
+* The API exists in API Manager and is configured as either a basic endpoint, or a proxy endpoint.
+* The Mule application is configured to use Anypoint Platform credentials.
++
+The platform credentials give your application access to the API Configuration in API Manager. You must configure these credentials before starting the Mule runtime engine that executes your application. 
+* The autodiscovery element is configured in your Mule application. 
++
+This XML element must point to the specific API in API manager which you want to pair to.
 
 == Proxy Endpoint vs Basic Endpoint
 
-If you configured a proxy endpoint, your autogenerated proxy will already be correctly configured with the Autodiscovery element, and if you choose to take advantage of the platform deployment capabilities, such as 1-Click deployment, the API Manager credentials will be automatically set for you as well. +
-Learn more autogenerated proxies in the xref:api-proxy-landing-page.adoc[API Proxy] section.
+If you configured a proxy endpoint, your autogenerated proxy is already correctly configured with the autodiscovery element. Additionally, if you want to take advantage of the platform deployment capabilities, such as 1-Click deployment, the API Manager credentials is automatically set for you. 
```

Comment:
Change "API Manager credentials is" to "API Manager credentials are"

Reply:
Done! Thanks!